readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://MykhailoLoniak.github.io/layout_stars/)
+- [TEST REPORT LINK](https://MykhailoLoniak.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the pr description
<img width="749" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/2f544978-532e-4fc9-87bc-78048b81f190">